src/node_options.h

```diff
@@ -193,6 +193,14 @@ class PerIsolateOptions : public Options {
 
 class PerProcessOptions : public Options {
  public:
+  // Options shouldn't be here unless they affect the entire process scope, and
+  // that should avoided when possible.
+  //
+  // When an option is used during process initialization, it does not need
+  // protection, but any use after that will likely require synchronization
+  // using the node::per_process::cli_options_mutex, typically:
+  //
+  //     Mutex::ScopedLock lock(node::per_process::cli_options_mutex);
```

Comment:
Should this be removed?

Reply:
Why?

Reply:
Sorry, at first glance with the extra indentation it looked like code that had been commented out.

Reply:
Or perhaps I should anticipate why you might ask - this has been done wrong multiple times, with a stream of fixup PRs. People are not finding that mutex buried in its .cc file (with no comment as to its purpose - https://github.com/nodejs/node/blob/e629366fc6c41b81e554fea0413ad1fac53adf36/src/node_options.cc#L28), so I suggest it be more prominently advertised, right over the variables that it protects. Or maybe you mean, shoudn't the whole object be made thread safe and the mutex removed? In which case, yes, but not in this PR.